From af1f2bb9a66a7ca4e73a5fc9c22631750a67f54e Mon Sep 17 00:00:00 2001
From: Paulo Alcantara <pc@cjr.nz>
Date: Fri, 6 Nov 2020 13:52:43 -0300
Subject: [PATCH 1/1] efi_loader: Add missing newline to log_{err,warning}

Add missing newline to log messages in efi_rng_register() otherwise
something like below would be shown

	Scanning disk virtio-blk#31...
	Found 2 disks
	Missing RNG device for EFI_RNG_PROTOCOLNo EFI system partition

Signed-off-by: Paulo Alcantara (SUSE) <pc@cjr.nz>
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/efi_rng.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/efi_loader/efi_rng.c b/lib/efi_loader/efi_rng.c
index a8a87007b6..8bdadad0a9 100644
--- a/lib/efi_loader/efi_rng.c
+++ b/lib/efi_loader/efi_rng.c
@@ -166,13 +166,13 @@ efi_status_t efi_rng_register(void)
 
 	ret = platform_get_rng_device(&dev);
 	if (ret != EFI_SUCCESS) {
-		log_warning("Missing RNG device for EFI_RNG_PROTOCOL");
+		log_warning("Missing RNG device for EFI_RNG_PROTOCOL\n");
 		return EFI_SUCCESS;
 	}
 	ret = efi_add_protocol(efi_root, &efi_guid_rng_protocol,
 			       (void *)&efi_rng_protocol);
 	if (ret != EFI_SUCCESS)
-		log_err("Cannot install EFI_RNG_PROTOCOL");
+		log_err("Cannot install EFI_RNG_PROTOCOL\n");
 
 	return ret;
 }
-- 
2.29.2

